community-members/camila-rangel-smith.md

```diff
@@ -0,0 +1,42 @@
+## Personas of the contributing members of _The Turing Way_
+
+* Name: **Camila Rangel Smith**
+* Pronouns: she/her
+* Date: 15 May 2020
+* Current involvement: Translation in Spanish
+
+### Bio (taken from The Alan Turing Institue)
```

Comment:
```suggestion
### Bio 
```

Reply:
Haha Good catch :D I felt compelled to mention the source but it is indeed a Bio.